cli/src/main/java/bisq/cli/CliMain.java

```diff
@@ -832,7 +832,6 @@ private static void printHelp(OptionParser parser, @SuppressWarnings("SameParame
             stream.format(rowFormat, "", "--security-deposit=<percent> \\", "");
             stream.format(rowFormat, "", "[--fee-currency=<bsq|btc>]", "");
             stream.format(rowFormat, "", "[--trigger-price=<price>]", "");
-            stream.format(rowFormat, "", "[--swap=<true|false>]", "");
```

Comment:
Doesn't this belong to `createoffer` help?

Reply:
I believe I accidentally deleted that line and put it back in.  It is in what has been merged.  It is also in `createoffer-help.txt`.